source/analyze-code.txt

```diff
@@ -121,6 +121,28 @@ information message displayed:
       .. figure:: /includes/images/builder-popup-photoshop.png
          :alt: Screenshot of builder expression in visual studio with information message displayed.
 
+The {+product+} supports builders variables tracking and composition. You can
```

Comment:
Suggestion: "builders variables" is reading confusing to me, would "builder variables" work just as well here?

```suggestion
The {+product+} supports builder variable tracking and composition. You can
```